pkg/netbox/api/ip_range_claim.go

```diff
@@ -129,13 +129,9 @@ func searchAvailableIpRange(availableIps *ipam.IpamPrefixesAvailableIpsListOK, r
 		return bytes.Compare(ips[i], ips[j]) < 0
 	})
 
-	for i := range ips {
+	for i := 1; i < len(ips); i++ {
 		currentIp := ips[i]
-
-		var previousIP net.IP
-		if i > 0 {
-			previousIP = ips[i-1]
-		}
+		previousIP := ips[i-1]
 
 		if i == 0 || areConsecutiveIPs(previousIP, currentIp) {
```

Comment:
`i` will never be 0 now, this condition can be simplified

Reply:
removed this check